lib/chef/config.rb

```diff
@@ -571,11 +571,12 @@ def self.env
     end
 
     def self.windows_home_path
-      env['SYSTEMDRIVE'] + env['HOMEPATH'] if env['SYSTEMDRIVE'] && env['HOMEPATH']
+      Chef::Log.deprecation("Chef::Config.windows_homw_path is now deprecated.  Please file an issue at https://github.com/chef/chef/issues if you're using it.  Consider using Chef::Util::PathHelper.home instead.")
```

Comment:
Also, why "Please file an issue at https://github.com/chef/chef/issues if you're using it." ? Seems like we either deprecate this or keep it around as a delegate method and don't deprecate it.